tests/network/hotplug_bridge.go

```diff
@@ -219,8 +223,14 @@ var _ = SIGDescribe("bridge nic-hotplug", func() {
 			hotPluggedVMI = verifyBridgeDynamicInterfaceChange(hotPluggedVMI, plugMethod)
 			Expect(libnet.InterfaceExists(hotPluggedVMI, "eth2")).To(Succeed())
 		},
-			Entry("In place", decorators.InPlaceHotplugNICs, inPlace),
-			Entry("Migration based", decorators.MigrationBasedHotplugNICs, migrationBased),
+			Entry("In place", libvmi.InterfaceDeviceWithBridgeBinding(secondHotpluggedIfaceName),
+				decorators.InPlaceHotplugNICs, inPlace),
+			Entry("Migration based", libvmi.InterfaceDeviceWithBridgeBinding(secondHotpluggedIfaceName),
+				decorators.MigrationBasedHotplugNICs, migrationBased),
+			Entry("In place link state down", interfaceInDownState(secondHotpluggedIfaceName),
+				decorators.InPlaceHotplugNICs, inPlace),
+			Entry("Migration based link state down", interfaceInDownState(secondHotpluggedIfaceName),
+				decorators.MigrationBasedHotplugNICs, migrationBased),
```

Comment:
Adding 4 test (counting the next change as well) for this is pretty extreme, don't you think?
Can you just add interfaces in the existing tests?